### PR TITLE
add admin dashboard template

### PR DESCRIPTION
**Pull request by at-hainguyen5**
## Description
Add Admin layout template.
## Reference Isuees
#2 

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-quantran2** [comment]
LGTM

File: public/css/dashboard.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
+body {
```
Comment (at-vienle) [inline public/css/dashboard.css]: locale this file in `public/css/admin/dashboard.css`

File: resources/views/admin/layouts/app.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Booking Clinic - @yield('title')</title>
+    <!-- Bootstrap core CSS -->
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
```
Comment (at-vienle) [inline resources/views/admin/layouts/app.blade.php]: if `app.css` contains only `bootstrap` then it's better to download original bootstrap4 to `public/css/bootstrap.css`

File: resources/views/admin/layouts/app.blade.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        @include('admin.layouts.sidebar')
+
+        <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-4">
+          @yield('content')
+        </main>
+      </div>
+    </div>
+
+    <!-- Icons -->
+    <script src="https://unpkg.com/feather-icons/dist/feather.min.js"></script>
```
Comment (at-vienle) [inline resources/views/admin/layouts/app.blade.php]: do you need it?

File: resources/views/admin/layouts/app.blade.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <ul class="navbar-nav px-3">
+        <li class="nav-item text-nowrap">
+          <a class="nav-link" href="#">Sign out</a>
+        </li>
+      </ul>
+    </nav>
+
+    <div class="container-fluid">
+      <div class="row">
+        @include('admin.layouts.sidebar')
```
Comment (at-vienle) [inline resources/views/admin/layouts/app.blade.php]: change directory to: `admin/layouts/partials/sidebar.blade.php`

File: resources/views/admin/layouts/app.blade.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <title>Booking Clinic - @yield('title')</title>
+    <!-- Bootstrap core CSS -->
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <script src="{{ asset('js/app.js') }}"></script>
+    <!-- Custom styles for this template -->
+    <link href="{{ asset('css/dashboard.css') }}" rel="stylesheet">
+  </head>
+
+  <body>
+    <nav class="navbar navbar-dark fixed-top bg-dark flex-md-nowrap p-0 shadow">
```
Comment (at-vienle) [inline resources/views/admin/layouts/app.blade.php]: separate navbar to partials view also

File: resources/views/admin/layouts/app.blade.php (hunk 3 above)
Comment (at-hainguyen5) [reply resources/views/admin/layouts/app.blade.php]: yes, i need it for display icon

**Pull request merged**